src/test/java/org/openelisglobal/address/AddressServiceIntegrationTest.java

```diff
@@ -0,0 +1,120 @@
+package org.openelisglobal.address;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openelisglobal.BaseTestConfig;
+import org.openelisglobal.address.service.AddressPartService;
+import org.openelisglobal.address.service.OrganizationAddressService;
+import org.openelisglobal.address.service.PersonAddressService;
+import org.openelisglobal.address.valueholder.AddressPK;
+import org.openelisglobal.address.valueholder.AddressPart;
+import org.openelisglobal.address.valueholder.OrganizationAddress;
+import org.openelisglobal.address.valueholder.PersonAddress;
+import org.openelisglobal.patient.PatientTestConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Mockito.when;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes = { BaseTestConfig.class, PatientTestConfig.class })
+@TestPropertySource("classpath:common.properties")
+@ActiveProfiles("test")
+public class AddressServiceIntegrationTest {
+
+    @Autowired
+    private AddressPartService addressPartService;
+
+    @Autowired
+    private OrganizationAddressService organizationAddressService;
+    @Test
+    public void testGetAll() {
+        List<AddressPart> addressPartsToInsert = new ArrayList<>();
+        AddressPart addressPart1 = createAddressPart("7", "AB Street", "102");
+        AddressPart addressPart2 = createAddressPart("8", "CD Street", "103");
+        addressPartsToInsert.add(addressPart1);
+        addressPartsToInsert.add(addressPart2);
+
+//        List<String> insertedIds = addressPartService.insertAll(addressPartsToInsert);
```

Comment:
Do you need this line 47 ? As well as line 50

Reply:
> Do you need this line 47 ? As well as line 50

I will remove it on the next push.